src/components/Button.js

```diff
@@ -1,20 +1,30 @@
-/* eslint-disable react/require-default-props */
-import React from 'react';
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({
-  name, wide = false, color = 'orange',
-}) => {
-  let sizeClass;
-  // eslint-disable-next-line no-unused-expressions
-  wide ? sizeClass = 'big-btn' : sizeClass = 'small-btn';
-  const colorClass = `${color}`;
-  // eslint-disable-next-line react/button-has-type
-  return <button className={`Button ${sizeClass} ${colorClass}`}>{name}</button>;
-};
+class Button extends Component {
```

Comment:
Components like this that do not have state should not be a class component, make them class components